docs/doxygen/Doxyfile.swig_doc.in

```diff
@@ -1318,7 +1311,7 @@ COMPACT_LATEX          = NO
 # by the printer. Possible values are: a4, letter, legal and
 # executive. If left blank a4 will be used.
 
-PAPER_TYPE             = a4wide
+PAPER_TYPE             = letter
```

Comment:
Why the paper format change?

Reply:
Doesn't matter, that SWIG file is now gone.

Reply:
@marcusmueller I believe it introduced a warning, so I changed it to match with the regular `Doxyfile.in`. This file is no longer necessary as @drmpeg points out.

Reply:
@japm48 Just FYI, Marcus created a new PR and merged all this today.https://github.com/gnuradio/gnuradio/commit/2025e70872dcbfa0039ec000ea11d98d34d6cf70